primitives/src/v1/mod.rs

```diff
@@ -57,6 +57,7 @@ pub use crate::v0::{ValidatorPair, CollatorPair};
 
 pub use sp_staking::SessionIndex;
 pub use sp_authority_discovery::AuthorityId as AuthorityDiscoveryId;
+pub use sp_consensus_slots::Slot;
```

Comment:
I just re-exported it so that we can use it in `cumulus` without depending on `sp-consensus-slots` (e.g. when we read from the storage key below). IMO if the storage key is important enough to have it exported, then it makes sense that the type of the value behind that key is also exported.